### DISCUSSION TITLE
Vaultwarden logo

### DISCUSSION THREAD

**Discussion by Gamodeur** [dani-garcia/vaultwarden#3051] [General]
Hello
I just made the last vaultwarden update and I just noticed that the bitwarden logos have been changed to the vaultwarden logos.
Do you have an easy way to restore the bitwarden logo?

**Comment by BlackDex** [comment]
Build the web-vault your self and omit the image replacement.
Or overrule those images via volume mounts.

May i ask why you want it changed? It is Vaultwarden you are running, not the official Bitwarden server.

**Reply by Gamodeur** [reply]
I don't think I can build an image.
I don't see how to cancel the images with a volume. Will there be no logo at all if I cancel the image editing?

I just wanted to let you know that I totally respect your work and have been using it for a while and it's great.
 But I'm used to the Bitwarden logo and I like it a lot even though I know it's not the official version.

**Reply by BlackDex** [reply]
There isn't an easy way, so either build the web-vault your self or have it as-is, is the only option currently.

We changed the logos because there is a lot of confusion. People reporting issues in regards to Vaultwarden at Bitwarden. We hope this way it will be less, and also more clear to people it's a different backend.

**Reply by Gamodeur** [reply]
OK it doesn't matter, I'll go with it.
I understand why the changes were made.

Thank you for your reply.
